libraries/botbuilder-dialogs-declarative/src/componentDeclarativeTypes.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @module botbuilder-dialogs-declarative
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DeclarativeType } from './declarativeType';
+import { ResourceExplorer } from './resources';
+
+/**
+ * Interface for registering declarative types.
+ */
+export interface ComponentDeclarativeTypes {
```

Comment:
Would it be helpful to add the type parameters from `DeclarativeType` to this type as well?

Reply:
The `getDeclarativeTypes` method would return a bunch of different `DeclarativeType`s, how can we add the type to `ComponentDeclarativeTypes`?